### PR TITLE
Enable PepDB module test

### PR DESCRIPTION
**Pull request by labkey-tchad**
#### Rationale
Not sure why this is disabled or how much work it would be to get it working.

#### Related Pull Requests
* https://github.com/LabKey/testAutomation/pull/838

#### Changes
* Remove `Disabled` category from `PepDBModuleTest`

### CODE REVIEW

File: src/org/scharp/atlas/pepdb/PepDBBaseController.java [hunk 1]
```diff
@@ -450,13 +452,11 @@ public void renderGridCellContents(RenderContext ctx, Writer out) throws IOExcep
...
-                    String href = "displayPeptide.view?" + PepDBSchema.COLUMN_PEPTIDE_ID + "=" + peptideId;
-                    out.write("<a href ='");
-                    out.write(href);
-                    out.write("' ");
-                    out.write("target='_self'>");
-                    out.write("P"+peptideId);
-                    out.write("</a");
+                    new Link.LinkBuilder("P" + peptideId).clearClasses()
+                            .href(new ActionURL(PepDBController.DisplayPeptideAction.class, getContainer())
+                                    .addParameter(PepDBSchema.COLUMN_PEPTIDE_ID, peptideId))
```
Comment (labkey-adam) [inline src/org/scharp/atlas/pepdb/PepDBBaseController.java:458]: `.target("_self")`?

File: src/org/scharp/atlas/pepdb/PepDBBaseController.java [hunk 2]
```diff
@@ -528,13 +528,11 @@ public void renderGridCellContents(RenderContext ctx, Writer out) throws IOExcep
...
-                    String href = "displayPeptidePoolInformation.view?" + PepDBSchema.COLUMN_PEPTIDE_POOL_ID + "=" + peptidePoolId;
-                    out.write("<a href ='");
-                    out.write(href);
-                    out.write("' ");
-                    out.write("target='_self'>");
-                    out.write("PP" + peptidePoolId);
-                    out.write("</a>");
+                    new Link.LinkBuilder("PP" + peptidePoolId).clearClasses()
+                            .href(new ActionURL(PepDBController.DisplayPeptidePoolInformationAction.class, getContainer())
+                                    .addParameter(PepDBSchema.COLUMN_PEPTIDE_POOL_ID, peptidePoolId))
```
Comment (labkey-adam) [inline src/org/scharp/atlas/pepdb/PepDBBaseController.java:535]: `.target("_self")`?

File: src/org/scharp/atlas/pepdb/PepDBBaseController.java [hunk 3]
```diff
@@ -605,15 +603,13 @@ public void renderGridCellContents(RenderContext ctx, Writer out) throws IOExcep
...
                     {
-                    out.write("<a href ='");
-                    out.write(href);
-                    out.write("' ");
-                    out.write("target='_self'>");
-                    out.write("PP" + parentPoolId);
-                    out.write("</a>");
+                        new Link.LinkBuilder("PP" + parentPoolId).clearClasses()
+                                .href(new ActionURL(PepDBController.DisplayPeptidePoolInformationAction.class, getContainer())
+                                        .addParameter(PepDBSchema.COLUMN_PEPTIDE_POOL_ID, parentPoolId))
```
Comment (labkey-adam) [inline src/org/scharp/atlas/pepdb/PepDBBaseController.java:613]: `.target("_self")`?

**Review by labkey-adam — CHANGES_REQUESTED** [review]
Add targets to links to maintain current behavior. Or maybe you removed them intentionally?

**Comment by labkey-jeckels** [comment]
Since Adam's already taken a look I won't review separately unless there's a good reason for me to

**Review by labkey-adam — APPROVED** [review]

**Comment by labkey-adam** [comment]
Agreed, no need for @labkey-jeckels to review

**Pull request merged**